Dockerfile

Dahlgren opened the pull request.

dscalzi commented:
I've never used docker before, could you provide some insight on this PR?

dscalzi commented on Dockerfile:
```diff
@@ -0,0 +1,5 @@
+
+EXPOSE 8080
+
+CMD [ "npm", "start" ]
```
Would this run `npm start` twice? Looks like the [8-onbuild][1] already runs `npm start`.

[1]: https://github.com/nodejs/docker-node/blob/39a5c8a3be7fff2ddc67a2e72919d0a3841b235f/8.9/onbuild/Dockerfile

Dahlgren replied:
Good catch, this would override the base images `CMD` instruction. I'll remove this one

dscalzi commented:
Also, should a [`.dockerignore`][1] file be included?

[1]: https://nodejs.org/en/docs/guides/nodejs-docker-webapp/#dockerignore-file

Dahlgren commented:
I'll add a `.dockerignore` to avoid copying some non needed files to the image

Jawnnypoo commented:
@Dahlgren Any word on this?

Dahlgren commented:
Changed to custom Dockerfile based on Node 8.9 image since the default `onbuild` image does not include the `package-lock.json`. I've also removed any non needed files and folders in the `.dockerignore` file.

dscalzi commented:
This can be merged so long as it has been tested and no one has any further comments.
